src/Engine/Screen.cpp

```diff
@@ -289,18 +250,17 @@ const SDL_Color *Screen::getPalette() const
 int Screen::getWidth() const
 {
 	int w, h;
-	SDL_GetWindowSize(_window, &w, &h);
+	SDL_GetRendererOutputSize(SDL_GetRenderer(_window), &w, &h);
 	return w;
 }
-
 /**
  * Returns the height of the screen.
  * @return Height in pixels
  */
 int Screen::getHeight() const
 {
 	int w, h;
-	SDL_GetWindowSize(_window, &w, &h);
+	SDL_GetRendererOutputSize(SDL_GetRenderer(_window), &w, &h);
```

Comment:
can't we use _renderer here instead of SDL_GetRenderer(_window) ?

Reply:
My first thought but "no matching function call" which I guess means it's the wrong type or null? delete _renderer and nulled further up the code, but I'm unfamiliar with the flow. I've bought a book.